CHANGES.rst

```diff
@@ -6,15 +6,15 @@ Changelog
 
 Breaking changes:
 
-- *add item here*
+    add item here
```

Comment:
When switching to `towncrier` and news snippets, the whole "unreleased" changelog section should be removed.

Reply:
Sure, thanks for the hint!